src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1

```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     RootModule        = 'Azs.Fabric.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.3.0'
```

Comment:
You don't have to change this.  Leave it as is.

Reply:
Talked to Bala, the change is needed.  I keep searching for Azs.Storage.Admin instead of Azs.Fabric.Admin